src/lib/components/ScriptureViewSofria.svelte

```diff
@@ -941,8 +992,21 @@ TODO:
                     {
                         description: 'Start Milestone',
                         test: () => true,
-                        action: ({ context }) => {
-                            // console.log('Start Milestone: %o', context.sequences[0].element);
+                        action: ({ context, workspace }) => {
+                            const element = context.sequences[0].element;
+                            if (element.subType === 'usfm:zvideo') {
+                                const id = element.atts['id'][0];
+                                const video = config.videos.find((x) => x.id === id);
+                                const divVideoBlock = createVideoBlock(
+                                    document,
+                                    video,
+                                    workspace.currentVideoIndex++
+                                );
+
+                                // TODO: Add it where we currently are in the document?
+                                // This isn't correct, but at least gets it into the document for now
+                                workspace.root.appendChild(divVideoBlock);
```

Comment:
@davidmoore1 I know this is not correct, but it at least gets the videos in the document.  How would I get them put in the current position in the document?

In the video books, the videos are separated by `\b` tags.
In a scripture book, someone could put it after a verse?